ormolu-live/deploy.sh

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
 set -eo pipefail
 [[ $BUILDKITE_BRANCH == "master" ]] && FLAGS="--prod" || true
-netlify deploy $FLAGS -d $(nix-build -A ormoluLive.website)
+ORMOLU_LIVE=$(nix-build -A ormoluLive.website)
+netlify deploy --alias=$(git log -1 --format="%H") -d $ORMOLU_LIVE
+netlify deploy $FLAGS -d $ORMOLU_LIVE
```

Comment:
Maybe we could replace `$FLAGS` by `--prod` directly since `deploy.sh` is only called when we are on `master` anyway?